projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/asm/customer360-promotion.e2e.cy.ts

```diff
@@ -111,7 +111,7 @@ context('Assisted Service Module', () => {
     it('should be able to sent customer coupon for customer coupon (CXSPA-3945)', () => {
       interceptPost(
         'claim_customer_coupon',
-        '/users/*/customercoupons/*/claim?*'
+        '/users/*/customercoupons/*/claim?*' //TODO check '/users/*/customercoupons/claim?*' instead when enable 'enableClaimCustomerCouponWithCodeInRequestBody' with the new Occ endpoint is available since Commerce 2211.28
```

Comment:
hi @rmch91  , not sure whether you are enjoying the holiday, hope not bother you. As I already disable the toggle to avoid break CI because of deployed commerce version issue. And I skip new added e2e test, but for the existed test cases, I left comments like this. Because when enable 'enableClaimCustomerCouponWithCodeInRequestBody', the case will failed because it need check new occ endpoints calling , so we need update the case here after enable 'enableClaimCustomerCouponWithCodeInRequestBody', do you think the left comments is enough for future update? or is there other steps for it? creating jira tickets to follow it? or other ideas to handle old cases after enable toggle.
Thanks.

Reply:
Hello @crisli001. Thank you. Let's try to do this way, I created a ticket to update CI server https://jira.tools.sap/browse/CXSPA-9190. 
You can create your own ticket to remove comments, change toggle and enable test, and setup a dependency in JIRA by linking this ticket I shared with you. 
After that you can track it. Once we will update CI server, you can enable all this things

Reply:
Hi @rmch91 , got it, created https://jira.tools.sap/browse/CXEC-47292 from our team side to follow the enable/update test and remove comments, thanks a lot.